### PR TITLE
Correct some outdate info

### PR DESCRIPTION
**Pull request by mariascharin**
Note that this is no comprehensive review of the material, only an
update of clearly outdated material.

### CODE REVIEW

**Comment by mariascharin** [comment]
@spacecowboy Please do a fact check of this change. Thanks!

File: working-with-data/guide-import-csv/guide-import-csv.adoc [hunk 1]
```diff
@@ -140,17 +140,20 @@ It combines multiple aspects in a single operation:
 
 ===== Memory Config
 
-* Make sure to have enough memory (at least 4G heap in `neo4j-wrapper.conf`) reserved for your Neo4j-Server or Neo4j-Shell (`export JAVA_OPTS="-Xmx4G"`)
-* Configure the memory mapping settings (Neo4j 2.1) according to your expected file sizes +
-  (Neo4j 2.1 `neo4j.properties` keep these ratios: `nodestore=100M, relationshipstore=2G, propertystore=500M, stringstore=500M`)
+* Make sure to set the heap size so a sufficient value.
```
Comment (spacecowboy) [inline working-with-data/guide-import-csv/guide-import-csv.adoc]: typo: `... so a sufficient value` -> `... to a sufficient value`

File: working-with-data/guide-import-csv/guide-import-csv.adoc [hunk 2]
```diff
@@ -140,17 +140,20 @@ It combines multiple aspects in a single operation:
...
+You do this by setting `dbms.memory.heap.initial_size` and `dbms.memory.heap.max_size` to at least 4G in `neo4j.conf`.
+If using cypher-shell, the memory is set by doing `export JAVA_OPTS="-Xmx4G"` before starting cypher-shell.
```
Comment (spacecowboy) [inline working-with-data/guide-import-csv/guide-import-csv.adoc]: I don't think this advice is necessary anymore. Cypher-shell does not use more memory during a csv import. This was probably a piece of advice intended for `neo4j-shell` when running in "offline" mode, where an embedded database was started. Cypher-shell does not have this functionality, it is all server-side.

The entire sentence can be removed IMO.

**Review by spacecowboy — COMMENTED** [review]
Just some comments

**Comment by mariascharin** [comment]
@jexp @mneedham Could one of you please review (and merge)? Thanks!

File: working-with-data/guide-import-csv/guide-import-csv.adoc [hunk 3]
```diff
@@ -140,17 +140,19 @@ It combines multiple aspects in a single operation:
...
+===== File-URLs
 
-* Make sure to use the right URLs esp. file URLs.+
-  On OSX and Unix use `file:///path/to/data.csv`, on Windows, please use `file:c:/path/to/data.csv`
-* Use the `bin/neo4j-shell` instead of the browser for better error messages and control +
-  by default it connects to a running server, but you can also use `bin/neo4j-shell -path import.db -config conf/neo4j.properties` for direct database directory access (when no server is running with that db).
+* Make sure to use the right URLs.
+In particular for file URLs: On OSX and Unix use `file:///path/to/data.csv`, and on Windows use a URL similar to: `file:c:/path/to/data.csv`.
+* Use `bin/cypher-shell` instead of the browser for better control.
+By default it connects to the database running on localhost but you can point it to a database anywhere over the network.
```
Comment (jexp) [inline working-with-data/guide-import-csv/guide-import-csv.adoc]: should also mention the import directory and that file urls are local to it
that is easy in the tarball distribution 
but a pain in neo4j-community (where the import directory resides *inside* of the database folder)

File: working-with-data/guide-import-csv/guide-import-csv.adoc [hunk 4]
```diff
@@ -184,7 +186,8 @@ You can find all scripts and data on https://github.com/nicolewhite/consumer_com
 
 === Super Fast Batch Importer For Huge Datasets
 
-With Neo4j 2.2, a new, massively parallel and scalable CSV importer was released as part of the product.
+`LOAD CSV` is great for importing small - medium sized data, i.e. up to the 10B records range.
```
Comment (jexp) [inline working-with-data/guide-import-csv/guide-import-csv.adoc]: rather 10M not 10B

File: working-with-data/guide-import-csv/guide-import-csv.adoc (hunk 3 above)
Comment (spacecowboy) [reply working-with-data/guide-import-csv/guide-import-csv.adoc]: @jexp I believe you mean neo4j-desktop here

**Comment by jexp** [comment]
Thanks a lot @mariascharin !

**Comment by mariascharin** [comment]
@jexp De nada. Thanks for addressing!

**Pull request merged**